h2o-algos/src/main/java/hex/rulefit/RuleFit.java

```diff
@@ -181,14 +183,17 @@ public void computeImpl() {
             try {
                 // linearTrain = frame to be used as _train for GLM in 2., will be filled in 1.
                 Frame linearTrain = new Frame(Key.make("paths_frame" + _result));
-                
+                // store train frame without bad columns to pass it to tree model builders
+                Frame trainRebalanced = new Frame(_train);
```

Comment:
I don't think "rebalanced" is a good name - it implies the dataset was rebalanced which is not always the case (rebalancing means redistributing the chunks)

it is (from RuleFit's point of view) - "adapted frame"

Reply:
5bd6d19